python/README.md

```diff
@@ -42,6 +37,7 @@ pip install -r test_requirements.txt
 ```
 
 Then run a test.
+
 ```
 pytest test/test_ops.py
 pytest test/ngraph/*
```

Comment:
I just noticed this, the `*` will break this command. Specify the path to directory, not a filesystem glob

Reply:
what directory? there are test files under this directory....i am giving the command for test_basic just as an example.

Reply:
The "test/" directory, instead of the "test/*" filename pattern. Just removing the * would be fine.

Reply:
Done.